prov/verbs/src/verbs_info.c

```diff
@@ -148,12 +148,7 @@ struct fi_ibv_rdm_sysaddr
 int fi_ibv_check_ep_attr(const struct fi_ep_attr *attr,
 			 const struct fi_info *info)
 {
-	if ((attr->type != FI_EP_UNSPEC) &&
-	    (attr->type != info->ep_attr->type)) {
-		VERBS_INFO(FI_LOG_CORE,
-			   "Unsupported endpoint type\n");
-		return -FI_ENODATA;
-	}
+	struct fi_ep_attr user_attr = *attr;
```

Comment:
Are we copying the attr here to skip protocol check in the util function? If that's the case the user_attr protocol is not set to UNSPEC in this function.

Reply:
Yep, you're right. Thanks for your attention.